libs/balances-and-allowances/README.md

```diff
@@ -0,0 +1,24 @@
+# Balances and Allowances
+
+The most of the lib logic is concentrated in the `BalancesAndAllowancesUpdater`.  
```

Comment:
Maybe just add one phrase before this one describing what this lib is for.

Reply:
Yes, sure!